openupgrade_scripts/scripts/fleet/18.0.0.1/post-migration.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2025 Open Net Sàrl
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+def _set_power_unit(env):
+    # if horspower is defined, set power_unit to horsepower
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE fleet_vehicle_model
+        SET power_unit='horsepower'
+        WHERE horsepower > 0
```

Comment:
```suggestion
        WHERE COALESCE(horsepower, 0) > 0
```

Reply:
```suggestion
        WHERE horsepower > 0 AND COALESCE(power,0) = 0
```

I think it's better to prioritize `power` in case both fields are filled.
<img width="693" height="114" alt="Selection_4892" src="https://github.com/user-attachments/assets/86242beb-1dea-4a68-9c4a-3a6a3868237a" />